DataAccessGeneration/DataAccessGeneration.csproj

```diff
@@ -7,8 +7,8 @@
     <Nullable>enable</Nullable>
     <PackageReadmeFile>readme.md</PackageReadmeFile>
     <PublishTrimmed>false</PublishTrimmed>
-    <AssemblyVersion>1.25</AssemblyVersion>
-    <FileVersion>1.25</FileVersion>    
+    <AssemblyVersion>1.27</AssemblyVersion>
+    <FileVersion>1.27</FileVersion>    
```

Comment:
skipping 1.26?

Reply:
1.26 was a real version but I failed to update the csproj file for it. (see readme.md)